lib/src/connector/connector.cc

```diff
@@ -47,7 +47,18 @@ Connector::Connector(const std::string& broker_ws_uri,
                      const std::string& client_crt_path,
                      const std::string& client_key_path,
                      const long& connection_timeout)
-        : broker_ws_uri_ { broker_ws_uri },
+        : Connector { std::vector<std::string>{broker_ws_uri}, client_type,
```

Comment:
Even if I prefer the above style, putting a few spaces around the init curly brackets will make it more consistent.